From 5493a93d4687ec7fec73324f6ea43b67e5424ae6 Mon Sep 17 00:00:00 2001
From: Allain Legacy <allain.legacy@windriver.com>
Date: Mon, 26 Feb 2018 08:03:04 -0600
Subject: [PATCH] spec: update package versioning for tis format

Updating the RPM version to align with TiS version format.

Signed-off-by: Allain Legacy <allain.legacy@windriver.com>
---
 SPECS/scapy.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SPECS/scapy.spec b/SPECS/scapy.spec
index 2ea1146..0fe347d 100644
--- a/SPECS/scapy.spec
+++ b/SPECS/scapy.spec
@@ -1,6 +1,6 @@
 Name:           scapy
 Version:        2.4.0
-Release:        2%{?dist}
+Release:        2.el7%{?_tis_dist}.%{tis_patch_ver}
 Summary:        Interactive packet manipulation tool and network scanner
 
 %global         gituser         secdev
-- 
2.7.4

